internal/template/template.go

```diff
@@ -35,7 +36,7 @@ type PatchData struct {
 	// Trial metadata
 	Trial metav1.ObjectMeta
 	// Trial assignments
-	Values map[string]int64
+	Values map[string]interface{}
```

Comment:
We cannot or we would break the template functions, we actually unroll the values before we put them in the map so the appropriate type is available reflectively in the templates.